README.md

```diff
@@ -1,82 +1,266 @@
+and [FauxAwsKmsSignatureServiceTest](./xrpl4j-core/src/test/java/org/xrpl/xrpl4j/crypto/signing/faux/FauxAwsKmsSignatureServiceTest.java)
+illustrate faux variants of a simulated external key provider that can also be used for further guidance.
+
+### Signing and Verifying Transactions
+
+The main interface used to sign and verify transactions
+is [SignatureService](./xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/crypto/signing/SignatureService.java),
+which has two concrete implementations: `BcSignatureService` and `BcDerivedKeySignatureService`. The first uses
+in-memory private key material to perform signing and validation operations, while the latter can be used to derive
+multiple private keys using a single entropy source combined with differing unique key identifiers (e.g., User Ids).
+
+#### Construct and Sign an XRP Payment:
+
+The following example illustrates how to construct a payment transaction, sign it using an in-memory private key, and
+then submit that transaction to the XRP Ledger for processing and validation:
+
+```java
+import org.xrpl.xrpl4j.crypto.core.keys.Seed;
+import org.xrpl.xrpl4j.crypto.core.keys.KeyPair;
+import org.xrpl.xrpl4j.crypto.core.keys.PrivateKey;
+import org.xrpl.xrpl4j.crypto.core.keys.PublicKey;
+import org.xrpl.xrpl4j.model.transactions.Address;
+import org.xrpl.xrpl4j.crypto.core.signing.SignatureService;
+import org.xrpl.xrpl4j.crypto.signing.bc.BcSignatureService;
+
+// Construct a SignatureService that uses in-memory Keys (see SignatureService.java for alternatives).
+SignatureService signatureService = new BcSignatureService();
+
+// Sender (using ed25519 key)
+Seed senderSeed = Seed.ed255519Seed();
+PrivateKey senderPrivateKey=senderSeed.derivePrivateKey();
+PublicKey senderPublicKey=senderPrivateKey.derivePublicKey();
+Address senderAddress=senderPublicKey.deriveAddress();
+
+// Receiver (using secp256k1 key)
+Address receiverAddress=Address.of("r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59");
+
+// Construct a Payment
+Payment payment=...; // See V3 ITs for examples.
+
+SingleSignedTransaction<Payment> signedTransaction=signatureService.sign(sourcePrivateKey,payment);
+SubmitResult<Payment> result=xrplClient.submit(signedTransaction);
+assertThat(result.result()).isEqualTo("tesSUCCESS");
 ```
 
 ## Development
+
+### Project Structure
+
+Xrpl4j is structured as a Maven multi-module project, with the following modules:
+
+- **xrpl4j-core
+  **: [![javadoc](https://javadoc.io/badge2/org.xrpl/xrpl4j-binary-codec/javadoc.svg?color=blue)](https://javadoc.io/doc/org.xrpl/xrpl4j-binary-codec)
+    - Provides Java objects which model XRP Ledger objects, as well as request parameters and response results for the
+      rippled websocket and JSON RPC APIs
+    - Also provides a Jackson `ObjectMapper` and JSON bindings which can be used to serialize and deserialize to and
+      from JSON
+    - Serializes the JSON representation of XRPL Transactions to the canonical binary format of the XRP Ledger
```

Comment:
Should we provide links to READMEs for each package?

Reply:
Hmmm...good question. Do you envision this going into the project's main README then?

Reply:
Fixed via cf2592bf80efa5a44b8583e605e302701783044c